### ISSUE TITLE
Retrying forever after parsing failed

### ISSUE THREAD

**Issue by singularity-s0** [merdok/homebridge-miot#278]
**Describe the bug**
The plugin for `hyd.airer.znlyj1` seems to have problems parsing the response from device. And though all features work, it keeps sending queries non-stop to the device.

**Expected behavior**
Since all expected feature works, we don't care about whether `target-position` is correctly parsed or not. But it should stop retrying after failing several times, instead of keep retrying forever.

**Your config.json**
```json
{
                    "name": "Clothes Airer",
                    "ip": "192.168.1.9",
                    "token": "(omitted)",
                    "deviceId": "361026402",
                    "model": "hyd.airer.znlyj1",
                    "pollingInterval": 10,
                    "deepDebugLog": true,
                    "micloud": {(using global config)},
                    "buzzerControl": true,
                    "ledControl": true,
                    "childLockControl": true,
                    "modeControl": true,
                    "motorControl": true
},
```

**Debug log**
```
[5/26/2022, 8:01:27 AM] [miot] [Clothes Airer] Device setup finished! Miot device ready!
[5/26/2022, 8:01:27 AM] [miot] [Clothes Airer] Connected to device: hyd.airer.znlyj1
[5/26/2022, 8:01:27 AM] [miot] [Clothes Airer] Doing initial property fetch.
[5/26/2022, 8:01:27 AM] [miot] [Clothes Airer] Splitting properties into chunks. Number of chunks: 1. Chunk size: 14
[5/26/2022, 8:01:27 AM] [miot] [Clothes Airer] Chunks:  [
 [
  "airer:on",
  "airer:status",
  "airer:fault",
  "airer:current-position",
  "airer:target-position",
  "airer:target-position2",
  "light:on"
 ]
]
[5/26/2022, 8:01:27 AM] [miot] [Clothes Airer] (MiCloud) Encrypted request https://api.io.mi.com/app/miotspec/prop/get - {"params":[{"did":"361026402","siid":2,"piid":17},{"did":"361026402","siid":2,"piid":4},{"did":"361026402","siid":2,"piid":1},{"did":"361026402","siid":2,"piid":3},{"did":"361026402","siid":2,"piid":15},{"did":"361026402","siid":2,"piid":16},{"did":"361026402","siid":3,"piid":1}]}
[5/26/2022, 8:01:27 AM] [miot] [Clothes Airer] Successfully saved device info!
[5/26/2022, 8:01:27 AM] [miot] [Clothes Airer] Error while parsing response from device for property airer:on. Response: {"did":"361026402","siid":2,"piid":17,"code":-704040003,"exe_time":0}
[5/26/2022, 8:01:27 AM] [miot] [Clothes Airer] Property airer:current-position value changed to ---> 1
[5/26/2022, 8:01:27 AM] [miot] [Clothes Airer] Error while parsing response from device for property airer:target-position. Response: {"did":"361026402","siid":2,"piid":15,"code":-704040003,"exe_time":0}
[5/26/2022, 8:01:27 AM] [miot] [Clothes Airer] Error while parsing response from device for property airer:target-position2. Response: {"did":"361026402","siid":2,"piid":16,"code":-704040003,"exe_time":0}

(and then repeats this forever)
```

**Comment by merdok** [comment]
It seems that the spec is wrong for the device. If you have a look here: http://merdok.org/miotspec and type the model then you can see that those properties actually does not exist. Not sure how this happend... Will correct that in the next update.

**Comment by singularity-s0** [comment]
> It seems that the spec is wrong for the device. If you have a look here: http://merdok.org/miotspec and type the model then you can see that those properties actually does not exist. Not sure how this happend... Will correct that in the next update.

This device recently had a firmware update. Could that be the cause?

**Comment by merdok** [comment]
Yeah it might be. Looking at the other Airer devices and saw that also there the spec has changed...

**Comment by merdok** [comment]
Should be fixed in version 1.2.2
